frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+      setHospitalAddress(address);
+    })();
+  }, [hospitalId]);
+
+  const handleInventoryChange  = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const updateList = [...inventoryList];
+    const target = e.target.getAttribute("data-index");
+    const dataIndex = parseInt(target !== null ? target : '0');
+    updateList[dataIndex]["available"] = parseInt(e.currentTarget.value);
+    setInventoryList(updateList);
+  }
+
+  const handleAddressChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const updateAddress = [...hospitalAddress];
+    const inputName = e.currentTarget.name;
+    const value = e.currentTarget.value;
+
+    if (inputName === 'zip') updateAddress[0][inputName] = parseInt(value);
+
+    if (inputName === 'city' || inputName === 'street' || inputName === 'country') 
+      updateAddress[0][inputName] = value;
+  
+    setHospitalAddress(updateAddress);
+  }
+
+  const updateInventoryList = () => {
+    // Update the inventory list
+  }
+
+  const updateHospitalAddress = () => {
+    // Update the hospital address
+  }
+
+  return (
+    <div className={style.containerDiv}>
+      <Form>
+        <h5>Address</h5>
+        <Form.Row>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="text" 
+              placeholder="street" 
+              name="street"
+              value={hospitalAddress[0] ? hospitalAddress[0].street : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="text" 
+              placeholder="city" 
+              name="city"
+              value={hospitalAddress[0] ? hospitalAddress[0].city : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="number" 
+              placeholder="zip" 
+              name="zip"
+              value={hospitalAddress[0] ? hospitalAddress[0].zip : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Form.Control as="select" 
+              name="country"
+              value={hospitalAddress[0] ? hospitalAddress[0].country : ''}
+              onChange={handleAddressChange}
+            >
+              <option value="India">India</option>
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Button 
+              variant="outline-primary" 
+              size="lg" 
+              block
+              onClick={updateHospitalAddress}
+            >
+              Update Info
+            </Button>{' '}
+          </Form.Group>
+        </Form.Row>
+      </Form>
+
+      <Form>
```

Comment:
I'm afraid this entire section shouldn't be here. Now that I look at the screenshot provided, it is there, but it probably is an artifact of the view being a mashup of the hospital admin view and the government official's view. Before you remove it, I'll ask Amey on Slack where it actually came from.